global-hooks/migrate/migrate_descheduler.go

```diff
@@ -98,12 +83,30 @@ func deschedulerConfigMigration(input *go_hook.HookInput, dc dependency.Containe
 			Namespace: "d8-system",
 		},
 		Data: map[string]string{"config": string(deschedulerConfigJSON)},
-	}, metav1.CreateOptions{})
-	if errors.IsAlreadyExists(err) {
-		input.LogEntry.Infof("CM already existis, skipping migration: %s", err)
-	} else if err != nil {
-		return err
 	}
 
+	input.PatchCollector.Create(cm, object_patch.UpdateIfExists())
```

Comment:
Do we need to create and update it afterward (after Deckhouse restarts)? What do we use as a flag that rescheduled already migrated?

Reply:
Yup, that's bad.

Here is my new solution. Create a CM once in the global hook. If the migration succeeds, annotate this CM and

Reply:
No objections to go with your solution 👍